apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -297,27 +293,26 @@ class ShareAllowedDialog extends React.Component {
                     <img style={styles.thumbnailImg} src={thumbnailUrl} />
                   </div>
                   <div>
-                    <p style={{fontSize: 20}}>{i18n.shareCopyLink()}</p>
-                    <input
-                      type="text"
-                      id="sharing-input"
-                      onClick={select}
-                      readOnly
-                      value={this.props.shareUrl}
-                      style={{cursor: 'copy', width: 450}}
-                    />
                     <button
                       type="button"
-                      id="share-dialog-copy-button"
+                      id="sharing-input-copy-button"
```

Comment:
Nit: it's a bit confusing that this is called `sharing-input-` even though it's not an input anymore. I'm assuming we were relying on this ID for logic in other places, which is why it wasn't worth changing?

Reply:
I changed this to unify the id with the one in `feedback.js` below but maybe I should've chosen `share-dialog-copy-button` instead. Let me see how easy it is to change them all at once